types/lib/project-manager.d.ts

```diff
@@ -0,0 +1,294 @@
+import SerializableClass from "abitbol-serializable";
+import type { Structure } from "./structure";
+
+interface WprjOptions {
+    type: string
+    metadataFormat: string
+    projectFormat: string
+    blobIndexFormat: string
+}
+
+export class ProjectManager extends SerializableClass {
+    __name__: "Project";
+
+    __init__(params: unknown): void;
```

Comment:
params: {
version,
mimetype,
fileExt
...
}

there are only 5 fields to define